src/ophyd_async/panda/_trigger.py

```diff
@@ -37,3 +43,38 @@ async def complete(self) -> None:
     async def stop(self):
         await self.seq.enable.set("ZERO")
         await wait_for_value(self.seq.active, False, timeout=1)
+
+
+@dataclass
+class PcompInfo:
+    start_postion: int  # start position in counts
+    pulse_width: int  # width of a single pulse in counts
+    rising_edge_step: int  # step between rising edges of pulses in counts
+    number_of_pulses: int
```

Comment:
```suggestion
    number_of_pulses: int #Number of pulses to send before disarming the PCOMP block. 0 means go on forever
```

Reply:
Or maybe

`#Number of pulses to send before the PCOMP block is disarmed. 0 means infinite`